packages/python/plotly/_plotly_utils/utils.py

```diff
@@ -227,7 +227,7 @@ def iso_to_plotly_time_string(iso_string):
 
 def template_doc(**names):
     def _decorator(func):
-        if sys.version[:3] != "3.2":
+        if not sys.version_info[:2] == (3, 2):
```

Comment:
should there be a `[:2]` in there? None of the changes in https://github.com/plotly/plotly.py/pull/2078/files include this index...

In fact, I think it might be nice to add this commit to https://github.com/plotly/plotly.py/pull/2078 and close this PR so we can merge it all at once, if you don't mind :)

Reply:
The `[:2]` is needed for `==` comparisons. It can be used but isn't required for `>`, `>=`, `<`, `<=` etc.

```pycon
>>> sys.version_info
sys.version_info(major=3, minor=8, micro=1, releaselevel='final', serial=0)
>>> sys.version_info[:2]
(3, 8)
>>> sys.version_info[:2] == (3, 8)
True
>>> sys.version_info == (3, 8)
False
>>> sys.version_info >= (3, 8)
True
>>> sys.version_info[:2] >= (3, 8)
True
>>> sys.version_info >= (3, 9)
False
>>> sys.version_info[:2] >= (3, 9)
False
```

---

Yep, will combine.

Reply:
> In fact, I think it might be nice to add this commit to #2078 and close this PR so we can merge it all at once, if you don't mind :)

Added, closing.